integ-test/src/test/java/org/opensearch/sql/legacy/MethodQueryIT.java

```diff
@@ -66,13 +67,13 @@ public void matchQueryTest() throws IOException {
    *
    * @throws IOException
    */
-  // todo
   @Test
+  @Ignore("score query no longer maps to constant_score in the V2 engine - @see org.opensearch.sql.sql.ScoreQueryIT")
```

Comment:
Move from legacy

Reply:
I've added the link.  We can remove these when we deprecate the legacy engine.